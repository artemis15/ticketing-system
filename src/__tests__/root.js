import React from 'react';
import { shallow , mount } from 'enzyme';
import Root from '../components/root';
import Login from '../components/login';

const setup = propOverrides => {
    const props = Object.assign({
        authorized: false
    }, propOverrides);

    const wrapper = mount(<Root {...props} />);

    return {
        props,
        wrapper
    }
};

describe('UI Tests', () => {
    it('Snapshot test', () => {
        const component = setup();
        expect(component).toMatchSnapshot();
    });
    it('Load login screen',()=>{
        const component = setup();
        const login = shallow(<Login />);
        console.log(component.wrapper);
        console.log(login);
        expect(true).toBe(true);
       // expect(component.wrapper).toMatch(login);

    });
});
