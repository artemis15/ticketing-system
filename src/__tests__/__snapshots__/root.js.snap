// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI Tests Snapshot test 1`] = `
Object {
  "props": Object {
    "authorized": false,
  },
  "wrapper": ReactWrapper {},
}
`;
